content/en/docs/Examples/services.md

```diff
@@ -662,6 +662,32 @@ service:
       icon: https://raw.githubusercontent.com/pterodactyl/panel/develop/public/assets/svgs/pterodactyl.svg
 ```
 
+## Radarr/Radarr
+Source: https://github.com/Radarr/Radarr
+
+- deployed_version - Requires an `API_KEY` which can be retrieved at `Settings/General/Security/API Key`
+```yaml
+service:
+    Radarr/Radarr:
```

Comment:
```suggestion
  Radarr/Radarr:
```
easy typo - too many spaces

Reply:
Not totally sure what to do in cases like this. Do I just tell you about it, or do I make a suggestion like this? 🤷

Reply:
Interesting, this suggestion feature is really useful. Guess I haven't been on GitHub enough. I've committed the change directly since you're going to squash on merge anyways. 👍